python/package/setup.py.in

```diff
@@ -1,13 +1,13 @@
 from distutils.core import setup
 
-setup(name='Optima',
+setup(name='optima',
       version='${PROJECT_VERSION}',
       description='The Python interface of the Optima library.',
       author='Allan Leal',
       author_email='allan.leal@erdw.ethz.ch',
       url='github.com/allanleal/optima',
       license='GNU LGPL v2.1',
       package_dir={'optima': ''},
-      packages=['optima', 'optima._extensions'],
```

Comment:
So far we do not have extensions for `optima`, so I have removed related unused codes.